clients/drcachesim/tests/invariant_checker_test.cpp

```diff
@@ -370,6 +440,8 @@ check_rseq()
             gen_marker(1, TRACE_MARKER_TYPE_RSEQ_ABORT, 2),
             gen_marker(1, TRACE_MARKER_TYPE_KERNEL_EVENT, 2),
             gen_marker(1, TRACE_MARKER_TYPE_KERNEL_EVENT, 4),
+            gen_instr(1, 10),
```

Comment:
Does the checker fail without these additions?  What if the trace did actually end before these were added?

Reply:
It does not fail.

As mentioned in the PR description, this is to verify that we don't raise a "signal handler return point incorrect" because of the apparent discontinuity between the post-signal instr and the instr just before the rseq-abort-causing signal.

See the paragraph with "Modified an existing invariant checker unit test to verify this scenario also." in the PR description.